File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_PhongBan_DinhSuat.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_PhongBan_DinhSuat } from './NS_PhongBan_DinhSuat.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_PhongBan_DinhSuatService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'PhongBan_Id', 'DinhSuat_Id'];

    List: NS_PhongBan_DinhSuat[] | undefined;
    ListFilter: NS_PhongBan_DinhSuat[] | undefined;
    FormData!: NS_PhongBan_DinhSuat;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_PhongBan_DinhSuat";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_PhongBan_DinhSuatService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_PhongBan_DinhSuatService : BaseService<NS_PhongBan_DinhSuat, INS_PhongBan_DinhSuatRepository>
    , INS_PhongBan_DinhSuatService
    {
    private readonly INS_PhongBan_DinhSuatRepository _NS_PhongBan_DinhSuatRepository;
    public NS_PhongBan_DinhSuatService(INS_PhongBan_DinhSuatRepository NS_PhongBan_DinhSuatRepository) : base(NS_PhongBan_DinhSuatRepository)
    {
    _NS_PhongBan_DinhSuatRepository = NS_PhongBan_DinhSuatRepository;
    }
    }
    }

